ValetWatchTest Extension/DetailController.h

```diff
@@ -0,0 +1,14 @@
+//
```

Comment:
Is this file (both the .h and .m) doing anything? Looks like they are boiler-plate. Can we remove them?